appinventor/components/src/com/google/appinventor/components/runtime/TableArrangement.java

```diff
@@ -66,7 +66,7 @@ public int Columns() {
    */
   @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_NON_NEGATIVE_INTEGER,
       defaultValue = "2")
-  @SimpleProperty(userVisible = false)
+  @SimpleProperty
```

Comment:
Don't make the row/column setters visible as there is no way to set the row/column of the views to fill those positions if they change.